fetch-events/event-data-template.njk

```diff
@@ -0,0 +1,5 @@
+{#- Template format for the generated event file  -#}
+// Auto Generated on {{ generationTime }}
+import { type EventItem } from "./types";
+
+export const events: readonly EventItem[] = {{ sortedEvents | dump(2) | safe }};
```

Comment:
The `safe` operator here stops the template renderer from escaping characters, which could theoretically lead to XSS issues.  However, the values are all returned from meetup which we can hopefully trust, and the existing raw-string method would've had the exact same problem anyway.